public/scripts/landing.js

```diff
@@ -4,7 +4,35 @@ class Landing extends React.Component {
 
   constructor(props) {
     super(props)
-    this.state = { gameId: '' }
+    this.state = { gameId: '',
+                   proposedID: '',
+                   invalidID: ''}
+    
+    txtFile = "../../wordlist.txt"          //maybe filepath needs fixing
+    file = new File(txtFile)
+    file.open("r") // open file with read access
+    wordArray = []
+    i = 0
+    while (!file.eof) {
+      // store each word in the array
+      wordArray[i] = file.readln()
+    }
+    file.close()
+    do{
+      this.state.proposedID = wordArray[Math.floor(Math.random() * wordArray.length)]
+      $.post({
+        contentType: 'application/json; charset=utf-8',
+        dataType: 'json',
+        data: JSON.stringify({ game_id: proposedID }),
+        url: `http://${DOMAIN}/new_game`,
+        success: this.onCreateGame
```

Comment:
in the event that teh request succeeds, you need to stop the while lop that you're in - otherwise you're sending too many unnecessary requests to the server.